test/shell-pipeline-tests/common.sh

```diff
@@ -74,3 +74,11 @@ function createImageRegistrySecret() {
     sed -i "s|{{ IMAGE_REGISTRY_USERNAME }}|${USERNAME}|" "$IMAGE_REGISTRY_SECRET_PATH"
     sed -i "s|{{ IMAGE_REGISTRY_PASSWORD }}|${PASSWORD}|" "$IMAGE_REGISTRY_SECRET_PATH"
 }
+
+function usePRBranchInPipelineRunIfPRCheck() {
+  local -r PIPELINE_RUN_FILE_PATH=$1
+
+  if [ -n $PULL_NUMBER ]; then
+    sed -i "s|value: \"main\"|value: \"pull/$PULL_NUMBER/head\"|" "$PIPELINE_RUN_FILE_PATH"
```

Comment:
Do we need to specific about which parameter's  `value: main` we are targeting to prevent from replacing globally in the instance that the `git-containerfile-repo` and `git-model-repo` are different.  

Right now, we can defer that use case to setting the requirement that the model repo and containerfile repo are the same

Reply:
@LaVLaS This is just for the tests, so I think for now it is good as we want to have everything from the pull request branch for the PR check. The example shows main for both, so I just override both to the pull request branch. But yes, in real world these can be different revisions. Is that OK?